lib/controllers/expense_controller.dart

```diff
@@ -4,8 +4,8 @@ import 'package:sun_flutter_capstone/models/model.dart';
 
 class ExpenseController {
   Future<List<Expense>> index(DateTime? startDate, DateTime? endDate) async {
-    DateTime paidStart = startDate ?? DateTime.now();
-    DateTime paidEnd = endDate ?? DateTime.now();
+    DateTime paidStart = startDate ?? DateTime.parse('2022-01-01 00:00:00');
+    DateTime paidEnd = endDate ?? DateTime.parse('2022-12-31 23:59:59');
```

Comment:
Why is the date very specific and not using current date?

Reply:
@inahsanjuan Before, If no start date and end date when fetching records, by default it will fetch the records of the current date. 

I changed the default date range value to the beginning day up to the end day of the year 2022 to fetch records within the whole year rather than single day. 

I will change the logic na lang to remove filtering by date range if start date and end date are not provided.